geometry_msgs/PointStamped mean
float64 likelihood
bool seeingBall
float64 meanU
float64 meanV

